hive/src/main/java/org/apache/iceberg/parquet/AdaptHiveParquetConversions.java

```diff
@@ -103,8 +103,15 @@ static Function<Object, Object> converterFromParquet(PrimitiveType parquetType,
         return ChronoUnit.MICROS.between(EPOCH, instant);
       };
     }
+
+    //string
+    if (icebergType == Types.StringType.get()) {
+      return binary -> StandardCharsets.UTF_8.decode(((Binary) binary).toByteBuffer());
```

Comment:
Is `UTF8` the only charset decoding string to binary in impala?

Reply:
yes